Composer/packages/client/src/components/GetStarted/GetStartedNextSteps.tsx

```diff
@@ -402,6 +402,7 @@ export const GetStartedNextSteps: React.FC<GetStartedProps> = (props) => {
             target="#luis"
             onDismiss={() => {
```

Comment:
Since there are two `onDismiss` actions with the same code, you should move this logic into a callback.

Reply:
Ahh missed that difference.